shell/platform/android/io/flutter/embedding/android/FlutterActivity.java

```diff
@@ -1129,14 +1130,14 @@ public boolean popSystemNavigator() {
 
   @Override
   public void updateSystemUiOverlays() {
-    if (delegate != null) {
+    if (delegate != null && delegate.isAttached()) {
       delegate.updateSystemUiOverlays();
```

Comment:
`updateSystemUiOverlays` could check this internally. This way there's no need to expose `isAttached` in the public API of the delegate.